.travis.yml

```diff
@@ -23,10 +25,16 @@ install:
     - popd && popd
     - git clone https://github.com/01org/TPM2.0-TSS.git
     - pushd TPM2.0-TSS
-    - git checkout 1.0
+    - git checkout 1.1.0_beta0
     - ./bootstrap && ./configure && make -j$(nproc) && sudo make install && popd
     - sudo ldconfig /usr/local/lib
-    - resourcemgr -sim -tpmhost 127.0.0.1 &
+    - git clone https://github.com/01org/tpm2-abrmd.git
+    - pushd tpm2-abrmd
+    - ./bootstrap && ./configure --with-dbuspolicydir=/etc/dbus-1/system.d && make -j$(nproc) && sudo make install && popd
```

Comment:
It mostly does. The default behavior according to GNU is for `${prefix}` to be set to `/usr/local` so w/o this `${dbuspolicydir}` ends up being under `/usr/local`. An alternative would be to use `--sysconfdir=/etc`. I went with using `--with-dbuspolicydir` directly since it makes the intent clear but I'm not particularly tied to either solution.